catalogs/qnl.yaml

```diff
@@ -8,7 +9,7 @@ sources:
       metadata_prefix: mods_no_ocr
       set:
     metadata:
-      data_path: qnl/british_library
+      data_path: qnl/british-library-combined
```

Comment:
When we harvest manually, the original records are split so that there is one in Arabic and one in English. I have a post harvest script that combines them into one file which is needed for mapping. I'm not sure what renaming it here means or if it matters but I assume it should be `british_library` which will get harvested as two files for each record and then get merged in post harvest.

Reply:
Essentially, this value needs to be whatever is used by dlme-transform to pick up the proper data, which is here: https://github.com/sul-dlss/dlme-transform/blob/main/config/metadata_mapping.json#L1572

So I think this change has to be `-combined`.